sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -101,11 +101,20 @@ extensions:
           make build-images
           docker tag openshift/origin-metrics-server:latest "openshift/origin-metrics-server:$( cat /data/src/github.com/openshift/aos-cd-jobs/ORIGIN_COMMIT )"
         fi
+    - type: "script"
+      title: "build the origin web console server container image"
```

Comment:
or bite the bullet and inherit from the built release + ecosystem

Reply:
don't have the time right now but i agree.